From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Thu, 13 Aug 2015 01:20:28 +0200
Subject: Call corosync directly in the service file

Modify init/corosync.service.in to call corosync directly instead of
through the upstream SysV initscript.
---
 init/corosync.service.in |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/init/corosync.service.in b/init/corosync.service.in
index d71a857..a1db617 100644
--- a/init/corosync.service.in
+++ b/init/corosync.service.in
@@ -5,9 +5,9 @@ Requires=network-online.target
 After=network-online.target
 
 [Service]
-ExecStart=@INITWRAPPERSDIR@/corosync start
-ExecStop=@INITWRAPPERSDIR@/corosync stop
-Type=forking
+EnvironmentFile=-/etc/default/corosync
+ExecStart=/usr/sbin/corosync -f $OPTIONS
+Type=simple
 
 # The following config is for corosync with enabled watchdog service.
 #
